src/lib/charts/chart-documentation/charts.examples.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { PieChart, BarChart } from '..';
 import { getCssVar } from '../../util/colors';
+import { ChartRefObject } from '../chart.helpers';
 
 export const PieChartExample1 = () => {
   return (
     <PieChart
       type="doughnut"
       tooltip={() => <div>Hello World</div>}
+      legend={(chartRef: ChartRefObject) =>
```

Comment:
Based on the component markup/type, isn't this function just receiving the `data`, not the `chartRef`. If we want to include the `chartRef` we would need to update the component and type (which maybe we should). In that case, we should probably pass the ref to custom tooltip components, as well.